project/Http4sPlugin.scala

```diff
@@ -115,15 +104,15 @@ object Http4sPlugin extends AutoPlugin {
   lazy val quasiquotes         = "org.scalamacros"          %% "quasiquotes"             % "2.1.0"
   lazy val reactiveStreamsTck  = "org.reactivestreams"       % "reactive-streams-tck"    % "1.0.0"
   lazy val scalacheck          = "org.scalacheck"           %% "scalacheck"              % "1.13.4"
-  def scalaCompiler(so: String, sv: String)     = so                      % "scala-compiler"            % sv
-  def scalaReflect(so: String, sv: String)      = so                      % "scala-reflect"             % sv
+  def scalaCompiler(so: String, sv: String) = so             % "scala-compiler"          % sv
+  def scalaReflect(so: String, sv: String) = so              % "scala-reflect"           % sv
   lazy val scalaXml            = "org.scala-lang.modules"   %% "scala-xml"               % "1.0.6"
-  def scalazCore(version: String)               = "org.scalaz"           %% "scalaz-core"               % version
-  def scalazScalacheckBinding(version: String)  = "org.scalaz"           %% "scalaz-scalacheck-binding" % version
-  def specs2Core(scalazVersion: String)         = "org.specs2"           %% "specs2-core"               % specs2Version(scalazVersion)
-  def specs2MatcherExtra(scalazVersion: String) = "org.specs2"           %% "specs2-matcher-extra"      % specs2Core(scalazVersion).revision
-  def specs2Scalacheck(scalazVersion: String)   = "org.specs2"           %% "specs2-scalacheck"         % specs2Core(scalazVersion).revision
-  def scalazStream(scalazVersion: String)       = "org.scalaz.stream"    %% "scalaz-stream"             % scalazCrossBuild("0.8.6", scalazVersion)
+  def scalazCore(szv: String)  = "org.scalaz"               %% "scalaz-core"             % szv
+  def scalazScalacheckBinding(szv: String)  = "org.scalaz"  %% "scalaz-scalacheck-binding" % szv
```

Comment:
Are you referring to the one extra space after scalazScalacheckBinding?  I was trying to align everything to where most of the dependencies were, in the soonest column possible.

I almost realigned the whole table.  I'm tempted.